City-of-Helsinki/paatokset-search
# PP-253: add env variables for connecting to proxy

Testing:

- Make sure your elastic container is up and running
- Git clone this repo: https://github.com/City-of-Helsinki/paatokset-elastic-proxy
- Follow installation instructions and see if you can get it up and running
- `cp .env.example .env` & add REACT_APP_ELASTIC_URL env variable to the newly created .env file (http://localhost:3000 should work)
- Start up React app using `PORT=3006 && npm start`
- Check that you get actual search results. Check network tab on browser and see that the app is using the proxy and not sending requests straight to elastic.
